deploy/helm/kubecf/assets/operations/instance_groups/credhub.yaml

```diff
@@ -24,16 +25,56 @@
       route_registrar:
         routes:
         - name: credhub
-          tls_port: 8844
+          tls_port: *credhub_port
           server_cert_domain_san: credhub.service.cf.internal
           registration_interval: 10s
           tags:
             component: credhub
           uris:
           - credhub.((system_domain))
-{{- end }}
 
-{{- if not .Values.features.credhub.enabled }}
+- type: replace
+  path: /instance_groups/name=credhub/jobs/-
```

Comment:
You're adding a container to a pod with an entrypoint that's not long running afaict.

Reply:
The idea was to also have a `trap` to clean up the security group on `EXIT`.
As we discussed on a chat, I'll change this to a job that is aware of the replicas.

Reply:
After re-assessing with Vlad, doing it as s job brings other complexities, like adding a service that performs load balancing to the replicas and then replacing the BOSH DNS aliases. That is a direction we don't want to take at this point. I'll follow my original idea here.

Reply:
:+1: — but I'm still exploring it as a job (`post-start` + `drain` scripts), using the same `POD_IP` mechanism you're using, to see how it goes.  That's _without_ the load balancing part — it'll just be co-located with each uaa / credhub instance.